public-ui/src/app/data-browser/charts/chart-version/chart-version-react.component.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { baseOptions, VERSION_STRATUM_MAP } from 'app/data-browser/charts/react-base-chart/base-chart.service';
+import * as highCharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+import * as React from 'react';
+
+interface State {
+    options: any;
+}
+
+interface Props {
+    versionAnalysis: any;
+    surveyVersionAnalysis: any;
+    selectedResult: any;
+}
+
+export class VersionChartReactComponent extends React.Component<Props, State> {
+
+  constructor(props) {
+    super(props);
+    this.state = {options: null};
+  }
+
+  componentDidMount() {
+      this.getChartOptions();
+  }
+
+  setChartOptions(analysisName: string, categories: any, series: any) {
```

Comment:
Not in this PR, but at some point we should go through all the chart components and see which functions we can make reusable and pull into a helper/utils file. Most of the prep functions for the chart options have a lot of common code, I believe this one is identical to `setCommonBioSexOptions` in `BioSexChartReactComponent`.

Reply:
Agree. After we pull out all the charts will work on moving common options to utils file.